File: templates/aspnetcore-csharp/enterprise/Controllers/HomeController.cs
using Microsoft.AspNetCore.Mvc;
using Enterprise.Services;
namespace Enterprise.Controllers {
    public class HomeController : Controller {
        [HttpGet("/")]
        public IActionResult Index() {
            return Content("<h1>Welcome to ASP.NET Core Enterprise Home!</h1>", "text/html");
        }
        [HttpGet("/api/data")]
        public IActionResult Data() {
            return Json(ApiService.GetData());
        }
    }
}

File: templates/symfony-php/enterprise/controllers/HomeController.php
<?php
namespace App\Controller;
use Symfony\Component\HttpFoundation\Response;
use App\Service\ApiService;
class HomeController {
    public function index() {
        return new Response('<h1>Welcome to Symfony Enterprise Home!</h1>');
    }
    public function apiData() {
        $data = ApiService::getData();
        return new Response(json_encode($data));
    }
}
